src/api/ingest/benthicpit_serializer.py

```diff
@@ -25,14 +25,15 @@ def benthic_attributes_choices():
 
 class BenthicPITCSVSerializer(CollectRecordCSVSerializer):
     protocol = "benthicpit"
-    observations_field = "data__obs_benthic_pits"
+    observations_fields = ["data__obs_benthic_pits"]
     ordering_field = "data__obs_benthic_pits__interval"
     additional_group_fields = CollectRecordCSVSerializer.additional_group_fields.copy()
     additional_group_fields.append("data__benthic_transect__label")
 
     header_map = CollectRecordCSVSerializer.header_map.copy()
     header_map.update(
         {
+            "Observation interval *": "data__obs_benthic_pits__interval",
```

Comment:
Was this really just overlooked previously? There's no similar field missing from fish belt?

Reply:
The observation interval was on the base serializer and I moved it out.  Yeah I think it was overlooked